grails-app/assets/javascripts/meriplan.js

```diff
@@ -783,7 +783,7 @@ function ReadOnlyMeriPlan(project, projectService, config, changed) {
     self.detailsLastUpdated = ko.observable(project.custom.details.lastUpdated).extend({simpleDate: true});
 
     self.isAgricultureProject = ko.computed(function () {
-        var agricultureOutcomeStartIndex = 4;
```

Comment:
This check needs to be done another way - by doing this you will break all existing agriculture projects using the RLP MERI plan